src/libs/Navigation/AppNavigator/Navigators/CentralPaneNavigator.js

```diff
@@ -11,23 +12,46 @@ const Stack = createStackNavigator();
 const url = getCurrentUrl();
 const openOnAdminRoom = url ? new URL(url).searchParams.get('openOnAdminRoom') : undefined;
 
+const commonOptions = {
+    headerShown: false,
+    title: 'New Expensify',
+
+    // Prevent unnecessary scrolling
+    cardStyle: styles.cardStyleNavigator,
+};
+
+function DummyScreen() {
+    return null;
+}
+
+const iframeScreens = [
+    SCREENS.HOME_OLDDOT,
+    SCREENS.EXPENSES_OLDDOT,
+    SCREENS.REPORTS_OLDDOT,
+    SCREENS.INSIGHTS_OLDDOT,
+    SCREENS.INDIVIDUAL_WORKSPACES_OLDDOT,
+    SCREENS.GROUPS_WORKSPACES_OLDDOT,
+    SCREENS.DOMAINS_OLDDOT,
+    SCREENS.WORKSPACE_OLDDOT,
+];
+
 function CentralPaneNavigator() {
     return (
         <FreezeWrapper>
-            <Stack.Navigator>
+            <Stack.Navigator screenOptions={commonOptions}>
                 <Stack.Screen
                     name={SCREENS.REPORT}
                     // We do it this way to avoid adding the url params to url
                     initialParams={{openOnAdminRoom: openOnAdminRoom ? 'true' : undefined}}
-                    options={{
-                        headerShown: false,
-                        title: 'New Expensify',
-
-                        // Prevent unnecessary scrolling
-                        cardStyle: styles.cardStyleNavigator,
-                    }}
                     component={ReportScreenWrapper}
                 />
+                {_.map(iframeScreens, (screen) => (
+                    <Stack.Screen
+                        key={screen}
+                        name={screen}
+                        component={DummyScreen}
```

Comment:
I know it may l may look weird but we need a screen that returns `null` here. 

We need to call the render method on the descriptor assigned to this route to have a proper URL in the address bar.

The actual IFrame is rendered in a different way in the custom navigator to make it reusable for every route.

Reply:
We could figure out a better name for this screen. Like `emptyScreen` or something.